ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -18,30 +12,58 @@ public static class Flow
         /// <returns><b>true</b> if <paramref name="value"/> is <b>false</b>; <b>false</b> if <paramref name="value"/> is <b>true</b>.</returns>
         public static bool InvertBoolean(bool value)
         {
-            // TODO: Fix this method so it passes all associated tests.
-            return value;
+            //Return the inverted boolean.
+            return !value;
         }
 
         /// <summary>
         /// Returns the correct <see cref="DriverAction"/> value for each <see cref="LightColor"/> using an if-else statement.
         /// </summary>
-        /// <param name="value">The <see cref="LightColor"/> for which to return a <see cref="DriverAction"/>.</param>
+        /// <param name="color">The <see cref="LightColor"/> for which to return a <see cref="DriverAction"/>.</param>
         /// <returns>The correct <see cref="DriverAction"/> value.</returns>
         public static DriverAction DriveSafelyIfElse(LightColor color)
         {
-            // TODO: Fix this method so it passes all associated tests. An if-else statement must be used to return the correct values.
-            return (DriverAction)24;
+            var driverAction = DriverAction.Unknown;
+
+            if (color == LightColor.Green)
+            {
+                driverAction = DriverAction.ProceedWithCaution;
```

Comment:
It's a stylistic choice whether to use a local variable and a single return statement or return within each if block, such as:

return DriverAction.ProceedWithCaution;

Some people prefer one option over the other, but I think both are useful depending on the situation.